### PR TITLE
MNT Standardise modules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/570

### CODE REVIEW

**Comment by GuySartorelli** [comment]
At least one CI job failed here

**Comment by emteknetnz** [comment]
Fixed

File: README.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-Allow members to manage and revoke access to multiple login sessions across devices.
-
 [![CI](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml)
-[![Code Quality](http://img.shields.io/scrutinizer/g/silverstripe/silverstripe-session-manager.svg?style=flat)](https://scrutinizer-ci.com/g/silverstripe/silverstripe-session-manager)
-[![Code coverage](https://codecov.io/gh/silverstripe/silverstripe-session-manager/branch/master/graph/badge.svg)](https://codecov.io/gh/silverstripe/silverstripe-session-manager)
-[![Version](http://img.shields.io/packagist/v/silverstripe/session-manager.svg?style=flat)](https://packagist.org/packages/silverstripe/session-manager)
-[![License](http://img.shields.io/packagist/l/silverstripe/session-manager.svg?style=flat)](LICENSE)
+[![Silverstripe supported module](https://img.shields.io/badge/silverstripe-supported-0071C4.svg)](https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list/)
 
-![CMS view](images/cms.png)
```
Comment (ChloeSartorelli) [inline README.md:11]: Why remove this?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (emteknetnz) [reply README.md:11]: Added back in

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**